clients/pd-send/main.go

```diff
@@ -0,0 +1,109 @@
+/*
+ *  This is a basic client that can plug into Nagios.
+ */
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"hash/crc32"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/streadway/amqp"
+)
+
+func failOnError(err error, msg string) {
+	if err != nil {
+		log.Fatalf("%s: %s", msg, err)
+	}
+}
+
+func getEnvWithDefault(env string, default_value string) string {
+	var v = os.Getenv(env)
```

Comment:
I considered that, but even if `LookupEnv` would return `("", true)` then I'd still want to set the default because that _clearly_ is not a valid AMQP string. 

(and yes, a To Do here is to do more checking on environment variables and arguments :-) )